odxtools/cli/_print_utils.py

```diff
@@ -1,11 +1,11 @@
 # SPDX-License-Identifier: MIT
 import re
 import textwrap
-from typing import Any, Callable, Dict, List, Optional, Union
+from typing import Any, Callable, Dict, List, Optional, Tuple, Union
 
 import markdownify
-from rich.padding import Padding
-from rich.table import Table
+from rich.padding import Padding as RichPadding
+from rich.table import Table as RichTable
```

Comment:
why?

Reply:
basically to avoid confusion with the `Table` class from odxtools. (similar story with `typing.Protocol` and `odxtools.Protocol`, with the difference that `rich.table.Table` feels pretty clunky to me so I used the `RichTable` alias.)